docs/synthetics/ssl-certificate-monitoring.md

```diff
@@ -6,10 +6,11 @@ Issues with SSL/TLS certificates doesn't only cause downtime of APIs and website
 * **Certificate Validation** - Validates the SSL certificates of the API/websites in every run
 * **Certificate Expiry** - Checks for the expiration of the certificates every day and alerts you 28, 14, 7 and 3 days before the expiry
 * **Certificate Change** - Checks for certificate change every 10 minutes and alerts you on detecting any changes, with a detailed change report
+* **Certificate Authority** - Checks Root/Intermediate certificate authorities and alerts you if they don't match with the expected authority
 
 Sematext Synthetics alerts you on the failure of these checks via the monitor's configured [alert notification hooks](../alerts/alert-notifications.md). Apart from these checks, Synthetics also provides an SSL certificate report, with details of all the certificates in the chain.
 
-Both the HTTP and Browser monitors perform these checks, except for Browser monitors with a configured User Journey script.
+Both the HTTP and Browser monitors perform these checks. Except certificate change detection is not available for Browser monitors.
```

Comment:
==> Both the HTTP and Browser monitors perform these checks, though certificate change detection is available only for HTTP monitors.